src/controllers/recipe.controller.ts

```diff
@@ -13,127 +13,125 @@ interface Pagination {
-        userId,
-        parsedName,
-        parsedSkip,
-        parsedLimit
-      );
+      if (admin) {
+        recipes = await recipeService.getRecipeList(
+          null,
+          parsedName,
+          parsedSkip,
+          parsedLimit
+        );
+      } else {
+        recipes = await recipeService.getRecipeList(
+          userId,
+          parsedName,
+          parsedSkip,
+          parsedLimit
+        );
+      }
+      return res.json(recipes);
+    } catch (err) {
+      return res.status(500).send();
     }
-    return res.json(recipes);
-  } catch (err) {
-    return res.status(500).send();
   }
-}
 
-// Reading recipe by id
-export async function readRecipeById(
-  req: Request,
-  res: Response
-): Promise<Response<RecipeDocument>> {
-  try {
-    const { id } = req.params;
+  // Reading recipe by id
+  public async readRecipeById(
+    req: Request,
+    res: Response
+  ): Promise<Response<RecipeDocument>> {
+    try {
+      const { id } = req.params;
 
-    const recipe = await recipeService.readRecipeById(id);
-    return res.json(recipe);
-  } catch (err) {
-    return res.status(500).send();
+      const recipe = await recipeService.readRecipeById(id);
+      return res.json(recipe);
+    } catch (err) {
+      return res.status(500).send();
+    }
   }
-}
 
-export async function updateRecipe(
-  req: Request,
-  res: Response
-): Promise<Response> {
-  try {
-    const { title, description, preparing, ingredients, url, id } =
-      req.body as RecipeWithId;
-    const { userId, admin } = req;
-    let result: boolean;
+  public async updateRecipe(req: Request, res: Response): Promise<Response> {
+    try {
+      const { title, description, preparing, ingredients, url, id } =
+        req.body as RecipeWithId;
+      const { userId, admin } = req;
+      let result: boolean;
 
-    if (admin) {
-      result = await recipeService.updateRecipe(id, null, {
-        title,
-        description,
-        preparing,
-        ingredients,
-        url,
-      });
-    } else {
-      result = await recipeService.updateRecipe(id, userId, {
-        title,
-        description,
-        preparing,
-        ingredients,
-        url,
-      });
-    }
-    if (result) {
-      return res.send();
-    } else {
-      // recipe doesn't belong to user or he sent the same copy and it wasn't modified
-      return res.status(400).send();
+      if (admin) {
+        result = await recipeService.updateRecipe(id, null, {
+          title,
+          description,
+          preparing,
+          ingredients,
+          url,
+        });
+      } else {
+        result = await recipeService.updateRecipe(id, userId, {
+          title,
+          description,
+          preparing,
+          ingredients,
+          url,
+        });
+      }
+      if (result) {
+        return res.send();
+      } else {
+        // recipe doesn't belong to user or he sent the same copy and it wasn't modified
```

Comment:
Avoid inserting such comments. It should be clear from the code itself what the method does. Leave comments for a very complicated logic (though usually it means it's not properly written / structured)